src/pykeen/experiments/boxe/abboud2020_boxe_fb15k37.yaml

```diff
@@ -0,0 +1,10 @@
+metadata:
```

Comment:
TODO: if we want the experimental configuration to be part of this PR, we should fill this one out, too.

Reply:
yes I was getting there! Thank you :)